### DISCUSSION TITLE
v2ray 是否可以支持配置热重载

### DISCUSSION THREAD

**Discussion by 0xffffharry** [v2fly/v2ray-core#1537] [Ideas]
比如geo资源文件，客户端的服务器配置，routing配置。可以实现很多功能，比如动态服务器配置（CDN不同地址不同端口，以免被间歇性阻断）。

**Comment by CrazyBoyFeng** [comment]
目前 v2ray 并不支持热重载，包括 linux 里的 service 都不支持 reload

**Comment by AkinoKaede** [comment]
No, sorry, it was difficult to achieve this.

**Comment by limit81995** [comment]
我也遇到了同样的问题，有时候想动态增加用户，或者动态修改配置，每次都要重启很麻烦。想通过shell 直接进行调用。

**Reply by qianbinbin** [reply]
`alias v2ray-restart='/usr/bin/systemctl restart v2ray*'`，可以写一个脚本监听配置目录
